packages/transformers/posthtml/src/PostHTMLTransformer.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+
+import posthtml from 'posthtml';
+import parse from 'posthtml-parser';
+import render from 'posthtml-render';
+import nullthrows from 'nullthrows';
+import semver from 'semver';
+
+const getPostHTMLConfig = async asset => {
+  let config = await asset.getConfig(
+    ['.posthtmlrc', '.posthtmlrc.js', 'posthtml.config.js'],
+    {
+      packageKey: 'posthtml'
+    }
+  );
+  if (!config && !asset.options.minify) {
+    return;
+  }
+
+  config = config || {};
+  const plugins = config.plugins;
+  if (typeof plugins === 'object') {
+    // This is deprecated in favor of result messages but kept for compatibility
```

Comment:
I could be wrong, but it looks like we won't need this anymore as `messages` covers the same thing, see #2966 and its changed files: https://github.com/parcel-bundler/parcel/pull/2966/files#diff-2dcba7b85fdbaf96d8c8a1e09d2c940eL39

cc @DeMoorJasper

Reply:
Yeah would be great to get this deprecated feature out in Parcel 2

Reply:
I will work to find a way to remove this. Wasn't care enough in to understand it when I grabbed that learning more and more about this now.

Reply:
Looks like the solution, is to just remove it 😅simple enough.